### ISSUE TITLE
failure to validate TMSs using morecantile

### ISSUE THREAD

**Issue by AndrewAnnex** [AndrewAnnex/planetcantile#3]
Currently, tests fail expectingly (see discussion in #2) because CRS URIs (e.g. http://www.opengis.net/def/crs/IAU/2015/70300) are not registered yet with the naming authority.
So code like `pyproj.CRS("http://www.opengis.net/def/crs/IAU/2015/70300")` returns an exception. 

The exception interested me because it is:
```
CRSError: Invalid projection: http://www.opengis.net/def/crs/IAU/2015/70300:
 (Internal Proj Error: proj_create: crs not found)
```
The `Internal Proj Error` part interested me, what also interested me was that calling `pyproj.CRS("IAU:70300")` _does not_ return an exception, it works just fine!:

```
In [5]: pyproj.CRS('IAU:70300')
Out[5]:
<Geographic 2D CRS: IAU_2015:70300>
Name: Titania (2015) - Sphere / Ocentric
Axis Info [ellipsoidal]:
- Lat[north]: Geodetic latitude (degree)
- Lon[east]: Geodetic longitude (degree)
Area of Use:
- undefined
Datum: Titania (2015) - Sphere
- Ellipsoid: Titania (2015) - Sphere
- Prime Meridian: Reference Meridian
```

This issue is just to keep track of this issue and work on it happening at https://github.com/opengeospatial/NamingAuthority/issues/212

**Comment by AndrewAnnex** [comment]
another thing to keep an eye out on is https://github.com/opengeospatial/2D-Tile-Matrix-Set, this pr https://github.com/opengeospatial/NamingAuthority/pull/206/ looks to be something that will need to also happen with our tilematrixsets

**Comment by AndrewAnnex** [comment]
Recently URIs like https://www.opengis.net/def/crs/IAU/2015/49900 started resolving however Proj still cannot make CRSs using them, switching all the URIs to URNs however does work and now #5 is passing tests

**Comment by AndrewAnnex** [comment]
theoretically fixed as of https://github.com/OSGeo/PROJ/pull/3569 which will be available in PROJ 9.2.0. Intermediate fix is to use URNs instead although they aren't technically valid tms 1.0 which requires URIs (maybe 2.0 is more flexible?)
